pkg/query-service/app/querier/querier.go

```diff
@@ -100,7 +100,7 @@ func (q *querier) execClickHouseQuery(ctx context.Context, query string) ([]*v3.
 		points := make([]v3.Point, 0)
 		for pointIdx := range series.Points {
 			point := series.Points[pointIdx]
-			if point.Timestamp > 0 {
+			if point.Timestamp >= 0 {
```

Comment:
`Timestamp` will be zero where it no timestamp column. We don't want to discard that point.